src/Microsoft.OpenApi.Readers/V2/OpenApiOperationDeserializer.cs

```diff
@@ -137,22 +139,33 @@ public static OpenApiResponses LoadResponses(ParseNode node)
             return domainObject;
         }
 
-        private static OpenApiRequestBody CreateFormBody(List<OpenApiParameter> formParameters)
+        private static OpenApiRequestBody CreateFormBody(ParsingContext context, List<OpenApiParameter> formParameters)
         {
             var mediaType = new OpenApiMediaType
             {
                 Schema = new OpenApiSchema
                 {
-                    Properties = formParameters.ToDictionary(k => k.Name, v => v.Schema)
+                    Properties = formParameters.ToDictionary(
+                        k => k.Name,
+                        v =>
+                        {
+                            var schema = v.Schema;
+                            schema.Description = v.Description;
+                            return schema;
+                        }),
+                    Required = formParameters.Where(p => p.Required).Select(p => p.Name).ToList()
                 }
             };
 
+            var consumes = context.GetFromTempStorage<List<string>>("operationconsumes") ??
```

Comment:
It is worth noting that this will import a valid V2 document correctly, but it will also create an invalid DOM if the media type is not a valid one.  Validating this is going to be tricky, so I'm ok if we punt on checking for that.

Reply:
Agree. It’s a bit complicated to add “validation logic” into the reader at this point. I’d rather wait for your PR to be completed and we can retroactively clean-up these validation grey areas later.